### PR TITLE
Mobile navigation menu with JavaScript

### PR DESCRIPTION
**Pull request by tkz96**
Created the mobile navigation menu with the help of JavaScript.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  navMenu.classList.toggle('active');
+});
+
+document.querySelectorAll('.nav-item').forEach(n => n.addEventListener('click', () => {
+    hamburger.classList.remove('active');
+    navMenu.classList.remove('active');
+}));
```
Comment (thecodechaser) [inline script.js]: - [ ]  Great job working on the functionality but there is a little issue, When the user clicks any of the mobile menu options, the mobile menu should disappear. Currently, this functionality isn't working, Please fix it so the mobile menu disappears when we click on any of the mobile-menu options.
- You can create events for the menu options so when there is a click on any of the options, We set our mobile menu `display: none`

Comment (thecodechaser) [inline script.js]: - [ ] After opening the mobile menu when I switch to the desktop screen the mobile menu is still on the screen, Please fix it so when we switch to the desktop screen, the Mobile menu should disappear.
- You can do that in your media queries so when we cross the min-width we can set our mobile menu `display: none`

File: css/styles.css [hunk 2]
```diff
@@ -338,22 +398,40 @@ textarea {
...
     padding: 0 155px;
   }
 
-  .app-header ul {
+  .nav-menu {
     display: flex;
     list-style: none;
-    gap: 32px;
+    background-color: none;
+  }
```
Comment (thecodechaser) [inline css/styles.css]: - [ ] Good job working on the styles but still there is a little issue. You are supposed to stick to the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as much as possible. Your design is a little different from the template. Please reduce the gap between your mobile menu options, Place some right margin to your close X button and make sure it matches the design.

Your design

Screenshot (186)

Figma design

Screenshot (185)

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## Hi @tkz96 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a title and summary. ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gemmen29 — APPROVED** [review]
Hi @tkz96, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**